spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
how does this differ from expect-expect?

Reply:
@akegan this rule doesn't let you do:
```
if (someCondition) {
  expect(athing).toEqual(anotherThing)
}
```

expect-expect doesn't let you do:

```
expect(athing)
```

(notice that there's no .toEqual(), toHaveLegth, etc. assertion on that so it's not actually testing anything

Reply:
I actually meant to ask about the difference with valid-expect so you somehow managed to answer my question with your first response!